### ISSUE TITLE
Example file

### ISSUE THREAD

**Issue by ElmoGeeraerts** [ElmoGeeraerts/intro_machine_learning_using_python#2]
I don't really understand what I have to put in the example file. I don't really see how an example file with content in is applicable to my code. Unless the excel files generated during the tutorial are deemed example files?

**Comment by montesmariana** [comment]
Yes, the example file is just meant to be the data that I can use to test / data you use to illustrate the module.

**Comment by montesmariana** [comment]
So, not so much the excel file generated _during_ the tutorial but the pre-existing data that you use to illustrate and that I can use to try out your code.
